README.md

```diff
@@ -0,0 +1,341 @@
+   requirement of preparing an explicit and strict API specifications, allows us
+   to easily answer the question: **if it doesn't work then who's fault is
+   that?**
+
+
+### Backward-compatibility
+
+ * As required by the rules of [semantic versioning](http://semver.org/), all
+   [breaking changes](https://en.wiktionary.org/wiki/breaking_change) (aka
+   backward-incompatible changes) SHOULD force the **major** API version number
+   to be increased (e.g. `1.3.7` should become `2.0.0`).
+
+ * API designers MUST take care of [backward-compatibility rules]
+   [backward-compatibility-rules], to facilitate predictable workflow for all
+   implementers.
+
+ * Once a document is [released][statuses], backward-incompatible changes
+   SHOULD be avoided. Once first implementations are deployed on production
+   servers, backward-incompatible changes MUST NOT occur, unless they cannot be
+   avoided.
+
+ * All non-trivial changes (especially the backward-incompatible ones) must go
+   through the proposal acceptance process, as described in the
+   [Rules for Submitting API Change Proposals](#change-proposals) section.
+
+
+### Git branches (and XML namespaces)
+
+ * All released specifications MUST be merged to a `stable-v<N>` branch, where
+   *N* is the major version number of the specification (ideally, *N* will
+   stay equal to `1` for the majority of all specifications).
+
+ * XML namespaces used throughout the project SHOULD "self-describe"
+   themselves. Namespace values SHOULD be valid GitHub URLs, pointing to
+   `stable-*` GitHub branches. Every XML document SHOULD conform to the
+   specification described in the `stable-*` branch given in its XML namespace.
+
+ * Major changes in API versions SHOULD lead to a new `stable-*` (e.g.
+   `stable-v2`) branch being created. If there is any XML namespace associated
+   with API, then this also means that a new XML namespace will be created (and
+   newer versions of XML documents will be required to use this namespace).
+
+ * Unreleased (draft) APIs also SHOULD use valid GitHub URLs for theirs XML
+   namespaces (most probably, URLs referring to the `master` branch).
+
+
+### Changelogs (aka Release Notes)
+
+ * Once a document is [released][statuses], every subsequent release of such
+   document MUST be accompanied by a *changelog*. All changes SHOULD be noted
+   in this changelog (and *backward-incompatible* changes should be
+   additionally highlighted).
+
+Please note, that this requirement is for **released** documents only, and it
+does not imply the need of having a detailed changelog made for every [change
+proposal][statuses] (although many change proposals MAY be accompanied by such
+change logs).
+
+
+### Preferred Data Formats
+
+We are inclined to use XML as our primary data exchange format. We might be
+persuaded towards JSON however. Further inquiry will be conducted and decisions
+will be released in future versions of this document.
+
```

Comment:
On 05/02/16 08:50, Wojciech Rygielski wrote:
> In README.md

> We already have an issue for that:
> erasmus-without-paper/general-issues#2
> <https://github.com/erasmus-without-paper/general-issues/issues/2> (I
> agree with what you said, but still, currently XML wins.)

If XML wins a democratic vote, at this day and age it is similarly easy
to process one or the other ;-)

-- 
Victoriano Giralt                             Central ICT Services
Systems Manager                               University of Malaga
+34952131415                                  SPAIN
==================================================================
Note: signature.asc is the electronic signature of present message
A: Yes.
> Q: Are you sure ?
>> A: Because it reverses the logical flow of conversation.
>>> Q: Why is top posting annoying in email ?